tests/e2e/browse-bills.spec.ts

```diff
@@ -0,0 +1,272 @@
+import { test, expect, Page, ElementHandle, Locator } from "@playwright/test"
+import { BillPage } from "./page_objects/billPage"
+
+test.describe("Search result test", () => {
+  test("should search for bills", async ({ page }) => {
+    const billpage = new BillPage(page)
+    await billpage.goto()
```

Comment:
I ended up changing how I did the navigation to the page so that I could put it into the `beforeEach`. I also noticed that you're using `uncheckAllFilters` in the filter tests, having the navigation in the `beforeEach` should load a new page so you shouldn't need to uncheck all filters

```
test.beforeEach(async ({ page }) => {
  await page.goto("http://localhost:3000/bills")
})
```

Reply:
Here is the link for your reference, it should go before the describe: https://github.com/codeforboston/maple/pull/1580/files#diff-defcf5d940ff8eed10dfa30e3aeab17a4efaf8f26c9aa5014654be88041c5ea7R4-R6